docs/source/conf.py

```diff
@@ -11,37 +11,92 @@
 # All configuration values have a default; values that are commented out
 # serve to show the default.
 
-
-try:
-  import fakeEnvironment
-except:
-  pass
-
 import datetime
 import os
 import sys
-import tempfile
+import subprocess
+
+sys.path.insert(0, ".")
 
 try:
-  diracRelease = os.environ[ 'DIRACVERSION' ]
-except KeyError:
-  diracRelease = 'integration'
+  import fakeEnvironment
+except ImportError:
+  pass
+try:
+  import fakeEnv
+except ImportError:
+  pass
```

Comment:
Yes, exactly. This is mocking all external packages. the fakeEnvironment.py was copied from the docs/Tools folder and updated for all ImportErrors during the documentation creation.